src/validation.h

```diff
@@ -311,11 +311,11 @@ void FlushStateToDisk();
 void PruneAndFlush();
 
 /** (try to) add transaction to memory pool **/
-bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
+bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const std::shared_ptr<const CTransaction> &tx, bool fLimitFree,
```

Comment:
Most of this PR was written before CTransactionRef was introduced, and I forgot to patch it up. I'm updating it now (and indeed, 3 cases were left).